src/visitable.cpp

```diff
@@ -11,29 +11,19 @@
 #include "game.h"
 
 template <typename T>
-item * visitable<T>::find_parent( item& it )
+item * visitable<T>::find_parent( item & )
 {
-    item *res = nullptr;
-    if( visit_items( [&]( item *node, item *parent, const tripoint * ){
-        if( node == &it ) {
-            res = parent;
-            return VisitResponse::ABORT;
-        }
-        return VisitResponse::NEXT;
-    } ) != VisitResponse::ABORT ) {
-        debugmsg( "Tried to find item parent using an object that doesn't contain it" );
-    }
-    return res;
+    return nullptr;
```

Comment:
Why is this reduce this to no-op? It's used for example in item_location::obtain()?